packages/calypso-e2e/src/lib/components/navbar-component.ts

```diff
@@ -22,22 +23,12 @@ export class NavbarComponent {
 		this.page = page;
 	}
 
-	/**
-	 * Wait for load state of the page.
-	 *
-	 * @returns {Promise<void>} No return value.
-	 */
-	private async pageSettled(): Promise< void > {
-		await this.page.waitForLoadState( 'load' );
-	}
-
```

Comment:
This `waitForLoadState` didn't actually do much that was useful, hence it is removed.